iceoryx_examples/icediscovery/iox_find_service.cpp

```diff
@@ -43,11 +43,11 @@ int main()
     {
         std::cout << "\n=========================================" << std::endl;
 
-        std::cout << "\nSearched for {'Radar', 'FrontLeft', 'Image'}. Found the following services:" << std::endl;
+        std::cout << "\nSearched for {'Radar', 'FrontLeft', 'Objects'}. Found the following services:" << std::endl;
         //! [search for unique service]
         serviceDiscovery.findService(iox::capro::IdString_t{"Radar"},
                                      iox::capro::IdString_t{"FrontLeft"},
-                                     iox::capro::IdString_t{"Image"},
+                                     iox::capro::IdString_t{"Objects"},
```

Comment:
With this change the C++ example is out of sync with the C example. Could you please make this change also in the c icediscovery example and adjust the explanation in the readme as well (also the integration tests for the c example).

But the easier approach would be to revert this change back to `Image` and you spare us reviewers and you some work.

Reply:
Thanks for the hint. I'll adapt corresponding c example and its integration tests as radar sensors typically don't send out images.